test/evergreen.yml

```diff
@@ -1783,6 +1792,31 @@ tasks:
             set -o verbose
 
             ./time_shift_test.sh /usr/local/lib/faketime/libfaketimeMT.so.1 0-1 2>&1
+  
+  - name: format-stress-sanitizer-test
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          posix_configure_flags: CC="clang -fsanitize=address" CFLAGS="-g -fno-omit-frame-pointer" --enable-diagnostic --with-builtins=lz4,snappy,zlib
```

Comment:
By convention, only those configure flags e.g. `--enable-diagnostic ----with-builtins=lz4,snappy,zlib` should be set in `posix_configure_flags` variable. Those environment variable settings should go to `configure_env_vars` variable.
Also, consider creating a new function (`compile-builtin`?) as I believe most stress jobs use the same builtin configure setting.

Reply:
Yes CC and CFLAGS should go to `configure_env_vars`. Not sure yet on creating a new task yet. Will followup on this idea once we start working on other stress tickets.